plugins/org.locationtech.udig.tool.select/src/org/locationtech/udig/tool/select/internal/AOILayerSelectionTool.java

```diff
@@ -131,7 +136,8 @@ private Envelope getBounds( MapMouseEvent e ) {
     /**
      * @see org.locationtech.udig.project.ui.tool.SimpleTool#onMouseDragged(org.locationtech.udig.project.render.displayAdapter.MapMouseEvent)
      */
```

Comment:
since you added `@Override` the javadoc with `@see` could be removed, could you adapt it as well?